source/blog/2016-04-27-baseURL.md

```diff
@@ -0,0 +1,104 @@
+---
+title: baseURL
+author: Nathan Hammond
+tags: Recent Posts, Ember CLI
+---
+
+The `baseURL` property in Ember CLI applications is often and tragically misunderstood. It is in fact misunderstood and buggy to the point that we have [at least 67 issues](https://github.com/ember-cli/ember-cli/search?q=baseURL&type=issues) referencing it. This guide is not about how to use it. **[Landing today in Ember CLI's canary channel, we're deprecating `baseURL`.](https://github.com/ember-cli/ember-cli/pull/5792)** Newly generated applications using this version of Ember CLI will not use this approach. This is the culmination of a lot of research and work by [Tobias Bieniek](https://github.com/Turbo87) and we're incredibly grateful for his efforts.
```

Comment:
`this version of Ember CLI` => `using Ember-CLI canary and starting in Ember-CLI 2.7`?

I think Ember-CLI is always dasherized.